purchases/src/main/kotlin/com/revenuecat/purchases/SyncPendingTransactionsHelper.kt

```diff
@@ -0,0 +1,114 @@
+package com.revenuecat.purchases
+
+import com.revenuecat.purchases.common.AppConfig
+import com.revenuecat.purchases.common.BillingAbstract
+import com.revenuecat.purchases.common.Dispatcher
+import com.revenuecat.purchases.common.LogIntent
+import com.revenuecat.purchases.common.caching.DeviceCache
+import com.revenuecat.purchases.common.log
+import com.revenuecat.purchases.identity.IdentityManager
+import com.revenuecat.purchases.models.StoreTransaction
+import com.revenuecat.purchases.strings.PurchaseStrings
+import com.revenuecat.purchases.strings.RestoreStrings
+import com.revenuecat.purchases.utils.Result
+
+internal class SyncPendingTransactionsHelper(
```

Comment:
`Sync` might be confusingly mistaken with `sync purchases`. Maybe `PostPendingTransactionsHelper` would be better?